fish.pc.in

```diff
@@ -1,6 +1,8 @@
 prefix=@prefix@
 datadir=@datadir@
 completionsdir=${datadir}/fish/vendor_completions.d
+functionsdir=${datadir}/fish/vendor_functions.d
+snippetsdir=${datadir}/fish/vendor_snippets.d
```

Comment:
We need to keep Homebrew in view here - these should all be separate configure-able variables with sensible defaults. I'm happy to fix that later as part of #2113.